### DISCUSSION TITLE
Create entry headless live preview graphQL problem

### DISCUSSION THREAD

**Discussion by sc-urs** [statamic/cms#6920] [Troubleshooting]
Hello, 

This github issue is the closest thing I could find that would describe the problem I'm having:

https://github.com/statamic/cms/issues/6134

Basically, I have a Next.JS frontend headlessly connected to Statamic CMS via GraphQL. 

The live preview function is connected properly and it works when editing already created entries.

The problem arises on the create entry page, where the live preview button is available but the post has not yet been saved nor published. 

There is no slug to which the live preview can point to in this case, because it has not yet been saved inside entries for graphQL to read, therefore the content I enter inside the entry has no way to show in the live preview.

What I'm wondering is if this is a feature that is not implemented yet or am I missing something here?

Thanks for your time!

**Comment by jasonvarga** [comment]
Feel free to just comment on the issue instead of opening a new discussion about the existing issue.
